en/docs/fhir/guides/smart-api.md

```diff
@@ -0,0 +1,139 @@
+# FHIR R4 SMART Configuration Service
+
+## Introduction
+
+This service provides API implementation of 
+[SMART Configuration API](https://www.hl7.org/fhir/smart-app-launch/#discovery-of-server-capabilities-and-configuration) 
+of a FHIR server.
+
+SMART defines a discovery document available at `.well-known/smart-configuration` relative to a FHIR Server Base URL, 
+allowing clients to learn the authorization endpoint URLs and features a server supports. This information helps clients to 
+direct authorization requests to the right endpoint, and helps clients construct an authorization request that the server 
+can support.
+
+!!! notes
+        Supported FHIR version is 4.0.1.
+
```

Comment:
Do the same as in https://github.com/wso2/open-healthcare-docs/pull/33/files#r1733971870 and add the ballerina tab

Reply:
Added